funcbench/main.go

```diff
@@ -50,38 +51,49 @@ type benchmarkTester struct {
 func main() {
 	// Show file line with each log.
 	log.SetFlags(log.Ltime | log.Lshortfile)
-	app := kingpin.New(filepath.Base(os.Args[0]), "benchmark result posting and formating tool.\n-i location of github hook file (even.json)")
+	app := kingpin.New(
+		filepath.Base(os.Args[0]),
+		"benchmark result posting and formating tool.",
+	)
 	app.HelpFlag.Short('h')
-	input := app.Flag("input", "path to event.json").Short('i').Default("/github/workflow/event.json").String()
+
+	owner := app.Flag("owner", "github owner or organisation name").Default("prometheus").Short('o').String()
+	repo := app.Flag("repo", "github repository name").Default("prometheus").Short('r').String()
+
+	prNumber := app.Flag("pr", "pull request ID").Short('p').Required().Int()
+	branch := app.Arg("branch", "branch name to test against").Required().String()
+	regex := app.Arg("regex", "regex to specify which benchmarks to run").Default(".").String()
+
+	raceEnabled := app.Flag("race", "branch name to test against").Default("false").Bool()
+	postComment := app.Flag("comment", "if post result as comment to original pull request, mainly for debugging").Default("true").Bool()
```

Comment:
I don't get this help ): What exactly this option does?

Reply:
maybe a `--dry-run` flag can substitute this case better?